PWGHF/DataModel/CandidateReconstructionTables.h

```diff
@@ -1252,6 +1252,25 @@ DECLARE_SOA_TABLE(DHadronRecoInfo, "AOD", "DHADRONRECOINFO",
                   aod::hf_correlation_d0_hadron::MDbar,
                   aod::hf_correlation_d0_hadron::SignalStatus);
 
+// definition of columns and tables for Ds-Hadron correlation pairs
+namespace hf_correlation_ds_hadron
+{
+DECLARE_SOA_COLUMN(DeltaPhi, deltaPhi, float);       //! DeltaPhi between Ds and Hadrons
+DECLARE_SOA_COLUMN(DeltaEta, deltaEta, float);       //! DeltaEta between Ds and Hadrons
+DECLARE_SOA_COLUMN(PtD, ptD, float);                 //! Transverse momentum of Ds
+DECLARE_SOA_COLUMN(PtHadron, ptHadron, float);       //! Transverse momentum of Hadron
+DECLARE_SOA_COLUMN(MD, mD, float);                   //! Invariant mass of Ds
+DECLARE_SOA_COLUMN(SignalStatus, signalStatus, int); //! Used in MC, Ds Signal
```

Comment:
The `signalStatus` column seems to be filled only with 0 or 1, right? In that case the `int` type is too large because it can be just `bool` instead.